src/ssh/cockpitsshrelay.c

```diff
@@ -306,7 +306,7 @@ challenge_for_knownhosts_data (CockpitSshData *data)
   gchar *ret = NULL;
   gchar *response = NULL;
 
-  response = challenge_for_auth_data ("host-key", data->outfd, NULL);
+  response = challenge_for_auth_data ("x-host-key", data->outfd, NULL);
```

Comment:
do we need to change the setting [here](https://github.com/stefwalter/cockpit/blob/e1f3a763e6e947f6a63b4e39198c1f3d8a0d0a55/src/ssh/cockpitsshrelay.c#L1169) as well?